src/display/annotation_layer.js

```diff
@@ -101,7 +101,9 @@ class AnnotationElementFactory {
           case "Ch":
             return new ChoiceWidgetAnnotationElement(parameters);
         }
-        return new WidgetAnnotationElement(parameters);
+        return new WidgetAnnotationElement(parameters, {
+          isRenderable: parameters.data.hasOwnCanvas,
+        });
```

Comment:
Looking at how all the other classes handle this, can we please add a suitable constructor in the `WidgetAnnotationElement`-class instead for consistency?
```javascript
  constructor(parameters) {
    const isRenderable = !!parameters.data.hasOwnCanvas;
    super(parameters, { isRenderable });
  }
```